pkg/sql/schema/sql_table.go

```diff
@@ -74,7 +93,7 @@ func (s SQLTable) DDL() DDL {
 		statements = append(statements, field.DDL().WriteBefore("  "))
 	}
 	for _, constraint := range s.Constraints {
-		statements = append(statements, constraint.DDL().WriteBeforeF("  constraint %s", pq.QuoteIdentifier(constraint.Name)))
+		statements = append(statements, NewDDL("  ").Merge(constraint.DDL()))
```

Comment:
this was also a workaround for the bug that was fixed in the `SQLTableConstraint.DDL`
No need for the workaround since the bug is fixed